nucypher/config/base.py

```diff
@@ -293,9 +296,14 @@ def deserialize(cls, payload: str, deserializer=json.loads, payload_label: Optio
             label = f"'{payload_label}' " if payload_label else ""
             raise cls.OldVersion(f"Configuration {label}is the wrong version "
                                  f"Expected version {cls.VERSION}; Got version {version}")
+
+        if 'keyring_root' in deserialized_payload:
```

Comment:
`keyring_root` became `keystore_path` as part of #2701. See https://github.com/nucypher/nucypher/commit/37a944df442f9df8ae8689408aa8a34472398287#diff-72cdbadc52be031579383783a54fab0b18127c72b9415b101270b55360be9f52R342

This was a concern I had regarding checking for specific keys, and what happens if they were changed. If we could somehow easily link them, that would be 👌 

Interesting that a test didn't fail, even something like that would be helpful.